From 9d81bf8d2e457540b8bd93859baef79a043c8482 Mon Sep 17 00:00:00 2001
From: Mike Hommey <mh@glandium.org>
Date: Wed, 20 Sep 2023 10:08:13 +0900
Subject: [PATCH 1/3] Revert "Fix test added in D150987 to account for
 different path separators which was causing the test to fail on Windows."

This reverts commit 2baf4a06ef06c51c2ef09f981f204983b0f8082c because it
depends on c649f29c24c9fc1502d8d53e0c96c3d24b31de1a which we revert.
---
 llvm/test/tools/llvm-nm/X86/line-numbers.test | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/llvm/test/tools/llvm-nm/X86/line-numbers.test b/llvm/test/tools/llvm-nm/X86/line-numbers.test
index e254df67f955..4acda8afb2a4 100644
--- a/llvm/test/tools/llvm-nm/X86/line-numbers.test
+++ b/llvm/test/tools/llvm-nm/X86/line-numbers.test
@@ -68,7 +68,7 @@
 # RUN: llvm-mc -g --filetype=obj %t/data-dwarf.s -triple=x86_64-pc-linux -o %t/data-dwarf.o
 # RUN: llvm-nm --line-numbers %t/data-dwarf.o | FileCheck %s --check-prefix=DATA-DWARF --match-full-lines --implicit-check-not={{.}}
 
-# DATA-DWARF: 0000000000000000 D defined_data /tmp{{\\|/}}tmp.c:1
+# DATA-DWARF: 0000000000000000 D defined_data /tmp/tmp.c:1
 
 #--- main.s
 .text
-- 
2.41.0.3.g1cb8d410ac

